src/server_manager/ui_components/outline-server-view.html

```diff
@@ -628,7 +628,13 @@ <h3>[[localize('data-limits-dialog-title')]]</h3>
     Polymer({
       is: "outline-server-view",
       created: function() {
-        this.bytes = require("bytes");
+        this.byteSize = require("byte-size");
+        this.byteSize.defaultOptions({
+          units: "metric",
```

Comment:
It seems all you need is this change, and revert the conversion changes below.

Reply:
Done